plugins/home/api-report.md

```diff
@@ -264,10 +294,40 @@ export type VisitListenerContextValue = {
 
 // @public
 export interface VisitsApi {
-  listUserVisits(queryParams?: VisitsApiQueryParams): Promise<Visit[]>;
-  saveVisit(saveParams: VisitsApiSaveParams): Promise<Visit>;
+  list(queryParams?: VisitsApiQueryParams): Promise<Visit[]>;
+  save(saveParams: VisitsApiSaveParams): Promise<Visit>;
+}
+
+// @public
+export class VisitsApiFactory implements VisitsApi {
+  protected constructor({
+    randomUUID,
+    limit,
+    retrieveAll,
+    persistAll,
+  }: VisitsApiFactoryOptions);
```

Comment:
note how when you do parameter unpacking directly in the method signature, it becomes part of the public API and leads to churn. You may want to just do eg `options: VisitsApiFactoryOptions` and unpacking inside methods to keep the reports clean for public things

Reply:
Fixed on 081fd8d7fac3427802046358ed074615fe19da01